conf/defaults.ini

```diff
@@ -1057,9 +1057,11 @@ prefix = prod.grafana.%(instance_name)s.
 #################################### Grafana.com integration  ##########################
 [grafana_net]
 url = https://grafana.com
+api_url = https://grafana.com/api
```

Comment:
WDYT about skipping the `[grafana_net.api_url]` and just providing `[grafana_com.api_url]`?

Reply:
Sounds good to me!